stock_move_actual_date/security/stock_move_actual_date_security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="group_actual_date_editable" model="res.groups">
+        <field name="name">Editable Actual Date</field>
+        <field name="category_id" ref="base.module_category_hidden" />
+        <field name="implied_ids" eval="[(4, ref('stock.group_stock_user'))]" />
```

Comment:
```suggestion
```
I think we can remove this to be a bit more flexible. Or does this group make no sense without this?

Reply:
I think it is better to remove because we can't also add another group in this module (eg, `group_mrp_user`).